simulation.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: page
+title: Second analysis steps
+subtitle: Generating signal decays
+minutes: 60
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Understand how a signal decay sample is produced in the LHCb framework
+> * Produce generator level Monte Carlo, print the decay tree and produce nTuples
+> * Read a DecFile and understand what it produces, including generator level cuts
+> * Filter a simulated sample to reduce disk space needed
+
+## What is Gauss?
+
+The LHCb simulation framework which steers the creation of simulated events and interfaces to multiple external applications. Most commonly, an event is created via the following procedure:
+
+1. The `ProductionTool` (Pythia, GenXicc, ...) generates an event with the required signal particle. Either by generating minimum bias events until a matching particle is found or by ensuring one is produced in every event.
+2. The signal particle is decayed using the `DecayTool` (EvtGen) to the desired final state, all remaining unstable particles are decayed independently.
+3. The signal and its decay products might be required to pass generator level cuts implemented as a `CutTool`.
+4. Particles are transported through the detector simulation.
+
+> ## Things to remember {.callout}
+>
+> 1. The detector simulation is the **__by far__** most time consuming step (minutes). So make sure your generator cuts remove events you cannot possible reconstruct or select later on. Additional options are available to increase the speed, please talk to your MC liaisons!
+> 2. The generator cuts are only applied to the signal that was forced to decay to the specific final state. _Any_ other true candidate is not required to pass.
+> 3. The number of generated events refers to the number entering step 4 above, so those passing the generator level cuts. __Not__ the number of events produced by the `ProductionTool` in the first step.
+
+## Figuring out which option files to use and how to run Gauss
+
+Imagine you need to know the option files and software versions used for a simulated sample you have found in the bookkeeping, e.g.
+```
+/MC/2015/Beam2510GeV-2015-MagDown-Nu1.5-25ns-Pythia8/Sim09b/Trig0x4115014e/Reco15a/Turbo01aEM/Stripping22NoPrescalingFlagged/27163003/ALLSTREAMS.DST
+```
+First, find the ProductionID:
+![FindingProductionID](img/simulation_1.png)
+Search for this ID in the Transformation Monitor, right click the result and select "Show request". Right clicking and selecting "View" in the new window will open an overview about all the individual steps of the production with their application version and option files used.
+> Important: the order of the option files does matter!
```

Comment:
I think you need to add something extra to here to use the special callout block style.